Clinical-Genomics/trailblazer
# Mip7.1 dev

This PR lets TB switch MIP version from 6 to 7

How to test:

install on stage of the hasta: bash servers/resources/hasta.scilifelab.se/update-trailblazer-stage.sh MIP7.1-dev
Install cg-mip7 on hasta bash servers/resources/hasta.scilifelab.se/update-cg-stage.sh MIP7.1-dev
Perform the trailblazer-MIP7 validation
Expected outcome:
See validation document

Review:
- [x] code approved by @barrystokman 
- [x] validation executed by @b4ckm4n, @patrikgrenfeldt 
- [x] New key from master tested by @patrikgrenfeldt 
Thanks for filling in who performed the code review and the test!

This is major version bump because it has changes that are not backwards compatible
